formats/taxon/publisher/details.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false
```

Comment:
Perhaps `internal_name` would be useful here too https://github.com/alphagov/govuk-content-schemas/pull/156